web/content/docs/benchmarks/heat-transport-bhe/3D_BHE_GW_advection.pandoc

```diff
@@ -0,0 +1,68 @@
++++
+author = "Boyan Meng, Chaofan Chen, Haibing Shao"
+date = "2019-11-07T11:52:00+01:00"
+title = "BHE with groundwater advection"
+weight = 123
+project = "Parabolic/T/3D_BHE_GW_advection/BHE_GW_advection.prj"
+
+[menu]
+  [menu.benchmarks]
+    parent = "heat-transport-bhe"
+
++++
+
+{{< data-link >}}
+
+## Problem description
+When groundwater flow is present, advective heat transport in the soil matrix has to be considered. Hence, the governing equation for advective and conductive heat transport in porous media can be expressed (in 2D form) as follows:
+$$
+\begin{equation}
+\rho c \frac{\partial T}{\partial t} + u \rho_w c_w \frac{\partial T}{\partial x} - \lambda \left(\frac{\partial^2 T}{\partial x^2}+\frac{\partial^2 T}{\partial y^2}\right) = 0
+\end{equation}
+$$
+where $u$ denotes an uniform Darcy velocity in the $x$ direction. Assuming an infinite porous medium with uniform intitial temperature, the analytical solution for the ground temperature response to an infinite line source along the $z$ direciton is expressed as (Diao et al. (2004)):
+$$
+\begin{equation}
+\Delta T(x,y,t)=\frac{q_L}{4\pi\lambda}{\rm exp}\left[\frac{v_Tx}{2a}\right]\int_{0}^{v_T^2t/4a} \frac{1}{\psi}{\rm exp}\left[-\psi-\frac{v_T^2(x^2+y^2)}{16a^2\psi}\right] {\rm d}\psi
+\end{equation}
+$$
+in which $v_T=u\rho_w c_w/\rho c$ is the effective heat transport velocity (Molina-Giraldo et al. (2011)) and $q_L$ is the constant heat exchange rate per unit length. As a common practice (e.g. Eskilson (1987) and Diao et al. (2004)), the BHE can be approximated by a line source. In cases where the BHE penetrates the entire depth of the 3D porous medium, the above analytical solution can be applied to solve the spatio-temporal distribution of the induced ground temperature.
+
+## Model Setup
+The geometry of the model is illustrated in Figure 1. The depth of the model domain is 15 m with an areal extent of 80 m x 80 m. The BHE is 1U-type and is represented by a straight line located at $x=0$ m and $y=30$ m. Since the groundwater flow is in $y$ direction, the mesh was intentionally extended downstream of the BHE, so that the boundary effects on the ground temperature distribution can be neglected even for the long-term simulation. Detailed parameters for the soil heat transport model can be found in the following table.
```

Comment:
Long lines are difficult to read. Please limit the text to 80 characters.

Reply:
Above you wrote: "uniform Darcy velocity in the $x$ direction", here the groundwater flow is in $y$ direction. Isn't this a contradiction?

Reply:
> Long lines are difficult to read. Please limit the text to 80 characters.

The text lines have been limited to less than 80 characters except for the table.


> Above you wrote: "uniform Darcy velocity in the $x$ direction", here the groundwater flow is in $y$ direction. Isn't this a contradiction?

Thank you for pointing out this issue. There was indeed some confusion. "The velocity is in the x-direction" is only an assumption for the given form of the analytical solution. In the benchmark, however, the velocity is assumed in the y-direction and therefore the analytical solution has to be adapted accordingly. To make things clear, the documentation file has been changed--the velocity in the governing equation is now modified as a vector with two components. The wording of other appearances of the velocity was also changed.